src/modules/auth/cookies.service.ts

```diff
@@ -10,11 +9,6 @@ export class CookiesServices {
       sameSite: 'none',
       secure: true,
     };
-    const { family, major } = useragent.parse(userAgent) || {};
```

Comment:
linter complains about `userAgent` not used :)

Reply:
@artursudnik Yes! I thought of removing the `userAgent` parameter, but then left it as is so that if any validation needs to be added we can use it. Do you think changing the signature would be good?

Reply:
we can always add it back. My personal preference is to keep the code clean, to avoid confusion later.